MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.h

```diff
@@ -31,7 +31,8 @@
 // MarkupsToModel includes
 #include "vtkSlicerMarkupsToModelModuleMRMLExport.h"
 
-static const char* MARKUPS_ROLE = "markupsToModelRoleNode";
+static const char* INPUT_MARKUPS_ROLE = "markupsToModelInputMarkups";
```

Comment:
As it's declared static, the symbols won't be visible in any other files, but it would be nicer to put them into the cxx file.